lib/ldap.js

```diff
@@ -93,7 +93,8 @@ function initializeConnection () {
   }
 
   if (!nconf.get('ANONYMOUS_SEARCH_ENABLED')) {
-    connection.bind(nconf.get("LDAP_BIND_USER"), LDAP_BIND_CREDENTIALS, function(err) {
+    ldapBindCredentials = nconf.get('LDAP_BIND_PASSWORD') || crypto.decrypt(nconf.get('LDAP_BIND_CREDENTIALS'));
```

Comment:
Are we still only decrypting once now or are we doing this again and again? Maybe this should check if `ldapBindCredentials` has been set already?

Reply:
initializeConnection() is called only when creating the connection but I can add that check too